cloudberry/zion/src/main/scala/edu/uci/ics/cloudberry/zion/model/impl/QueryValidator.scala

```diff
@@ -169,6 +181,9 @@ object QueryValidator {
   }
 
   def validateTextRelation(relation: Relation, values: Seq[Any]): Unit = {
+    if (relation == Relation.isNull || relation == Relation.isNotNull) {
+      return
+    }
```

Comment:
Similar to the previous comments, if the `null` logic is handled in a unified place, we should not need to validate it here.